packages/desktop-client/src/components/ManageRules.js

```diff
@@ -222,6 +229,14 @@ export function ConditionExpression({
   );
 }
 
+function describeSchedule(s, payee) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** `s` is very vague‏
```suggestion
function describeSchedule(schedule, payee) {
```